project-base/storefront/components/Blocks/ConfirmationPage/ConfirmationPageContent.tsx

```diff
@@ -0,0 +1,24 @@
+import { ReactElement } from 'react';
+
+type ConfirmationPageContentProps = {
+    heading: string;
+    content?: string;
+    AdditionalContent?: ReactElement;
+};
+
+export const ConfirmationPageContent: FC<ConfirmationPageContentProps> = ({ heading, content, AdditionalContent }) => {
+    return (
+        <div className="mt-16 mb-10 flex flex-col items-center justify-center lg:mt-16 lg:mb-24 lg:flex-row">
+            <div className="mb-0 w-40 lg:mr-32">
+                <img alt={heading} src="/public/frontend/images/sent-cart.svg" />
+            </div>
+            <div>
+                <div className="h1 mb-3">{heading}</div>
+                {!!content && (
+                    <div className="text-center lg:text-left" dangerouslySetInnerHTML={{ __html: content }} />
+                )}
+                {!!content && !!AdditionalContent && AdditionalContent}
```

Comment:
```suggestion
                {!!content && AdditionalContent}
```

No need for check since undefined will not be shown in the DOM.

Reply:
Done and moved under the previous condition.

Reply:
Nice